### ISSUE TITLE
Sanitize inputs

### ISSUE THREAD

**Issue by immortalx** [benawad/graphql-ts-server-boilerplate#9]
Does yup or something else you are using sanitizes the user inputs?
I'm using https://github.com/leizongmin/js-xss and it works very well.

**Comment by benawad** [comment]
I believe react does

**Comment by immortalx** [comment]
Even if react does, the api should take care of it since it can be consumed by other clients. Again, i'm not sure if yup or even if typeorm does this but it's a fair concern for a production ready system.

**Comment by benawad** [comment]
yeah it's worth doing if there are other clients that don't use react

**Comment by immortalx** [comment]
This article is from 2016 https://medium.com/node-security/the-most-common-xss-vulnerability-in-react-js-applications-2bdffbcc1fa0
It seems it affects mostly SSR apps.
Anyway it was just a concern i had and i actually do it in my api, but i don't use yup.
